_licenses/lal-1.3.txt

```diff
@@ -0,0 +1,215 @@
+peut augmenter pour l'avantage de tous.
+
+C'est la raison essentielle de la Licence Art Libre : promouvoir et protéger
+ces productions de l'esprit selon les principes du copyleft : liberté d'usage,
+de copie, de diffusion, de transformation et interdiction d'appropriation
+exclusive.
+
+Définitions :
+
+Nous désignons par « œuvre », autant l'œuvre initiale, les œuvres conséquentes,
+que l'œuvre commune telles que définies ci-après :
+
+L'œuvre commune : Il s'agit d'une œuvre qui comprend l'œuvre initiale ainsi
+que toutes les contributions postérieures (les originaux conséquents et les
+copies). Elle est créée à l'initiative de l'auteur initial qui par cette licence
+définit les conditions selon lesquelles les contributions sont faites.
+
+L'œuvre initiale : C'est-à-dire l'œuvre créée par l'initiateur de l'œuvre
+commune dont les copies vont être modifiées par qui le souhaite.
+
+Les œuvres conséquentes : C'est-à-dire les contributions des auteurs qui participent
+à la formation de l'œuvre commune en faisant usage des droits de reproduction,
+de diffusion et de modification que leur confère la licence.
+
+Originaux (sources ou ressources de l'œuvre) : Chaque exemplaire daté de l'œuvre
+initiale ou conséquente que leurs auteurs présentent comme référence pour
+toutes actualisations, interprétations, copies ou reproductions ultérieures.
+
+Copie : Toute reproduction d'un original au sens de cette licence.
+
+   1- OBJET.
+
+Cette licence a pour objet de définir les conditions selon lesquelles vous
+pouvez jouir librement de l'œuvre.
+
+   2. L'ÉTENDUE DE LA JOUISSANCE.
+
+Cette œuvre est soumise au droit d'auteur, et l'auteur par cette licence vous
+indique quelles sont vos libertés pour la copier, la diffuser et la modifier.
+
+      2.1 LA LIBERTÉ DE COPIER (OU DE REPRODUCTION).
+
+Vous avez la liberté de copier cette œuvre pour vous, vos amis ou toute autre
+personne, quelle que soit la technique employée.
+
+      2.2 LA LIBERTÉ DE DIFFUSER (INTERPRÉTER, REPRÉSENTER, DISTRIBUER).
+
+Vous pouvez diffuser librement les copies de ces œuvres, modifiées ou non,
+quel que soit le support, quel que soit le lieu, à titre onéreux ou gratuit,
+si vous respectez toutes les conditions suivantes :
+
+1. joindre aux copies cette licence à l'identique ou indiquer précisément
+où se trouve la licence ;
+
+2. indiquer au destinataire le nom de chaque auteur des originaux, y compris
+le vôtre si vous avez modifié l'œuvre ;
+
+3. indiquer au destinataire où il pourrait avoir accès aux originaux (initiaux
+et/ou conséquents).
+
+Les auteurs des originaux pourront, s'ils le souhaitent, vous autoriser à
+diffuser l'original dans les mêmes conditions que les copies.
+
+      2.3 LA LIBERTÉ DE MODIFIER.
+
+Vous avez la liberté de modifier les copies des originaux (initiaux et conséquents)
+dans le respect des conditions suivantes :
+
+1. celles prévues à l'article 2.2 en cas de diffusion de la copie modifiée
+;
```

Comment:
I imagine the license text must be copied from https://github.com/spdx/license-list-data/blob/7364995d38090ad40b8aa54d99b322ac8730d2e1/text/LAL-1.3.txt ... that's definitely the right place to start, but it looks like it has some quirks (ie bugs in the SPDX XML markup and/or XML-to-text rendering); this line stands out, as do the doublespaced list items.

Do you know of a plain text version (French) that has been published by artlibre.org and/or is in wide use to compare against? I guess the examples requested above will be good indicators.

Reply:
The strange spacing is in the original text, which is http://artlibre.org/
It's remains at https://spdx.org/licenses/LAL-1.3.html and in their repositories
There is also https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Licence_Art_Libre

Wikipedia says that ": ; ? and ! are always preceded by a thin unbreakable space". I'm assuming full width spaces were used, and it's breaking that line. Elsewhere it mentions nonbreaking spaces for before and after those characters: https://en.wikipedia.org/wiki/French_orthography#Punctuation

Reply:
Yes, I saw that the spacing around punctuation is in the original. I meant the spacing between lines, and in the one line I'm highlighting here, oddly aggressive line wrapping just before punctuation. In any case, let's see what the full French text examples you find look like, and we'll see if any issues or PRs should be filed with SPDX about their text rendered version (which I very much want to be aligned with).

Reply:
For the punctuation spacing, I think either U+00A0 or U+202F might look better. Apparently the thin non-breaking space [only goes before the punctuation](https://forum.wordreference.com/threads/fr-space-before-a-colon-semicolon-question-mark-or-exclamation-point.54376/#post-16703459).

For the bigger problem of spacing between lines, it definitely seems like LAL-1.3.xml got thrown off somewhere, it looks like starting at line 47. There is also an (I think) unnecessary/redundant <p> element at line 122 and a missing <br> element at 165. For these reasons I went ahead and made a version, working from GPL-3.0.xml as a pattern. It's at spdx/license-list-XML#915